Makefile

```diff
@@ -140,25 +140,26 @@ verify-vendor:
 
 .PHONY: generate-kustomize
 generate-kustomize: bin/helm
-	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -n kube-system -s templates/clusterrole-attacher.yaml > ../../deploy/kubernetes/base/clusterrole-attacher.yaml
```

Comment:
i added -n kube-system, otherwise it seems like helm template uses whatever namespace your current context is :/. (might be rebase issue getting rid of it.)

Reply:
In this context, the rolebindings are the only thing that cares about the namespace since they need to set the namespace for the service accounts.  The rest of the resources get the namespace they will be created in from the kubectl command.  Unless I am missing the point.  I don't believe that any of the generated /base/ files have any namespace changes in them.  Let me know if there is something I am missing.

Reply:
yeah, it's only the bindings that i'm worried about. I added the arg to guarantee they stay at kube-system because I merged some change that inadvertently changed it and I had to change it back (obviously, I should not have merged it in first place)

Reply:
Okay, let me know if I missed a rolebinding.  I removed the others to make it clear that we aren't forcing a namespace into any of the other manifests.  If you think there is a compelling reason to put them back on the other resources I can but I didn't want others to be confused.

Reply:
makes sense. the right solution is to just review things better, not to force a namespace in the Makefile rule.